src/metabase/query_processor/streaming/common.clj

```diff
@@ -29,6 +30,10 @@
   Object
   (format-value [this] this)
 
+  ISeq
+  (format-value [this]
+    (into [] (map format-value this)))
```

Comment:
Out of curiosity, why do you prefer `(into [] (map format-value this))` to `(mapv format-value this)`? Also, if we keep `into`, I would write `(into [] (map format-value) this)`.

Reply:
no preference, forgot about `mapv`